### DISCUSSION TITLE
on change password get FORBIDDEN - You don't have permission to access this

### DISCUSSION THREAD

**Discussion by newathnow** [directus/directus#19892] [Feature Requests]
### Summary

Hey guys any ideas why i get this: 
{
    "errors": [
        {
            "message": "You don't have permission to access this.",
            "extensions": {
                "code": "FORBIDDEN"
            }
        }
    ]
}

Directus 10.2.0

PASSWORD_RESET_URL_ALLOW_LIST="https://domain.com/reset-passwrod"

Do i need to change some settings in Roles and Permissions? (but i am using admin should do the work no ?

### Basic Example

this is example how my express looks (i have used sdk for everything but reset password didnt worked with it and wrote with rest.

app.post("/reset-password", async (req, res) => {
  const { token, password } = req.body;
  let newPassword =  password;
  let accessToken = "my_adminoken";
  let finalURL = `https://directusbackend.com/auth/password/reset`;
  try {
    const response = await axios.post(finalURL, {
      token:token, password:newPassword}, {
      headers: {
        Authorization: `Bearer ${accessToken}`,
      },
    });
    const responsePass = response.data.data;
    res.status(200).json({ message: "Password reset successful." });
    
  } catch (error) {
    console.error(error);
    res.status(500).json({ error: "An error occurred during password reset." });
  }
});

### Motivation

NA

### Detailed Design

NA

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

NA

### Alternatives

NA

### Adoption Strategy

NA

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request. We rely on [Discord](https://directus.chat) for community Q&A 👍🏻
